### ISSUE TITLE
readme.md a bit messy

### ISSUE THREAD

**Issue by danmarsden** [efox12/Leaderboard-Plugin#11]
looks like a lot of stuff you've copied from a template somewhere? - would be good to tidy it up and make it specific to your plugin.

**Comment by calltri** [comment]
Yeah, pretty much everything above the "Modify the Multiplier/Points System" header is generic enough that you could apply it to more or less any plugin. You could arguably still keep it (I know it helped me at least) but it would be good to clearly label the uniquely leaderboard related information so people with experience can skip straight to it.
